eng/pipelines/corefx-base.yml

```diff
@@ -79,6 +84,10 @@ jobs:
         - ${{ if eq(job.submitToHelix, 'true') }}:
           - group: DotNet-HelixApi-Access
 
+        - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
+          - group: DotNet-Blob-Feed
+          - _dotnetFeedUrl: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
```

Comment:
Just as an FYI that this syntax that I suggested is not officially supported by azure devops for variables, but since we use arcade's template job.yml -- it parses the variables that come as objects and transforms them to the name-variable syntax that you had before:
```
- name: <name>
   value: <variable value>
```

https://github.com/dotnet/arcade/blob/master/eng/common/templates/job/job.yml#L103-L109